package.json

```diff
@@ -62,12 +64,18 @@
     "webpack-bundle-analyzer": "^2.9.0",
     "webpack-dashboard": "^1.0.0-6",
     "webpack-dev-server": "^2.8.2",
-    "webpack-merge": "^4.1.0"
+    "webpack-merge": "^4.1.0",
+    "@storybook/react": "^3.2.10",
+    "@storybook/addon-actions": "^3.2.10",
+    "@storybook/addon-links": "^3.2.10"
   },
-  "dependencies": {
+  "peerDependencies": {
     "react": "^15.6.1",
-    "react-dom": "^15.6.1",
-    "britecharts": "^2.0.1"
+    "react-dom": "^15.6.1"
```

Comment:
you're going to need to have `react` & `react-dom` as `devDependencies` as well.

Might as well use/support React 16 now right? Maybe in a follow-up diff

Reply:
Thanks! I will add them then.

Yeah, I would love to have this up to date with React, but it should go by hand with EDS, right?

Reply:
You're not using EDS here right? You shouldn't have to wait.

You may want to set the `peerDependencies` to be `>= 15` so that anyone on 15 or 16 can use it. Right now it's basically excluding everyone except the last version of 15 😄

Reply:
Thanks! Doing it on the next PR.